core/modules/xrdsvc/ChannelStream.cc

```diff
@@ -87,11 +87,10 @@ ChannelStream::append(char const* buf, int bufLen, bool last) {
     if(_closed) {
         throw Bug("ChannelStream::append: Stream closed, append(...,last=true) already received");
     }
-    //LOGF_INFO("last=%1% %2%" % last % makeByteStreamAnnotated("StreamMsg", buf, bufLen));
-    LOGF_INFO("last=%1% %2%" % last % util::prettyCharBuf(buf, bufLen, 10));
+    LOGS_DEBUG("last=" << last << " " << util::prettyCharBuf(buf, bufLen, 10));
```

Comment:
I wasn't sure how much I should be messing around with logging related to xrootd. I am happy to do that.